var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -336,7 +336,7 @@ def cmake_args(self):
             maxfields = maxfields << 1
         options.append('-DLegion_MAX_FIELDS=%d' % maxfields)
 
-        options.append('-DBUILD_MARCH:STRING={0}'.format(self.spec.architecture.target))
+        options.append('-DBUILD_MARCH:STRING=')
```

Comment:
@scothalverson for posterity, can we include a comment here saying that this is being done to disable the legion cmake's direct use of `-march` (and also `-mcpu`) so that spack's compiler wrappers can control the target architecture?

Reply:
Sure! I'll add it